pkg/module/filter.go

```diff
@@ -116,11 +118,16 @@ func (f *Filter) getAssociatedRule(path ...string) FilterRule {
 	return f.root.rule
 }
 
-func (f *Filter) initFromConfig() {
-	lines, err := getConfigLines(f.filePath)
+func (f *Filter) initFromConfig() error {
+	// Do not return an error if the file path is empty
+	// Do not attempt to parse it as well.
+	if f.filePath == "" {
+		return nil
+	}
 
-	if err != nil || len(lines) == 0 {
-		return
+	lines, err := getConfigLines(f.filePath)
+	if err != nil {
+		return err
 	}
 
 	for _, line := range lines {
```

Comment:
can you update getConfigLines to return an error of file.Close()?
another thing which comes to mind and can be addressed in different PR but i want to hear what you think: should we fail if config/filter file is not in the correct format? Now we ignore entries which does not start with `+` `-` or `D`

Reply:
yep! I can do that.
Let us do that in a separate PR.
And if we are thinking of ideas, it will be nice to have an integration test that checks for modules in one of the storages.
Also, update a simple filter.conf in the docs.